Sources/natalie/Parser.swift

```diff
@@ -34,7 +34,7 @@ struct Parser {
 
         output += header.description
         output += "import \(os.framework)\n"
-        for module in storyboards.lazy.flatMap({ $0.storyboard.customModules }) {
+        for module in Set(storyboards.lazy.flatMap { $0.storyboard.customModules }) {
```

Comment:
Yes, for multiple  storyboards with same imported module

Tested with and without

This one is the only one mandatory to fix the issue
The other one is for logic and to prevent future issue